routing_model/index.html

```diff
@@ -319,14 +320,34 @@ <h2>Deciding when to accept an incoming relocation</h2>
 
   <h2>Relocated node connection</h2>
   <div class=description>
-  <p>Manage node with NodeState=WaitingConnectionInfo and connection info request/response.<br/>
-    CandidateInfo is sent to NaeManager of the target interval<br/>
-    When we complete, we either stop responding to CandidateInfo RPCs, or we send NodeConnected RPC<br/>
+  <p>Manage node with NodeState=WaitingConnectionInfo and connection info request/response RPCs.<br/>
+    The candidate sends CandidateInfo to NaeManager of the target interval, so it will reach the managing section after split/merge.<br/>
+    This target interval address could be the middle address of the interval, but the full interval should not be overlapping two section after a split.<br/>
```

Comment:
This is a bit unclear to me. Could you explain what you mean by "the full interval shouldn't be overlapping" ?

Reply:
Replaced with:

and the interval should cover a range that will not be split if the section splits.